.env.integration

```diff
@@ -20,12 +20,12 @@ BLOCKSCOUT_SUBNETWORK_NAME="Integration"
 GETH_NODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/geth"
 # When upgrading change this to latest commit hash from the master of the geth repo
 # `geth $ git show | head -n 1`
-GETH_NODE_DOCKER_IMAGE_TAG="7a12221e9c7ca5a9f6a7904b0325d4ee275041bc"
+GETH_NODE_DOCKER_IMAGE_TAG="7fbd6f3574f1c1c1e657c152fc63fb771adab3af"
```

Comment:
I think integration is still using `7a12221e9c7ca5a9f6a7904b0325d4ee275041bc` for now

Reply:
> 7fbd6f3574f1c1c1e657c152fc63fb771adab3af

This is for the next deployment. Going forward `gs://env_config_files/integration` would contain what's deployed and `.env.integration` will contain what should be used for the next deployment.